### PR TITLE
#706: Migrate to scalafix patches

### PR DESCRIPTION
**Pull request by ysusuk**

### CODE REVIEW

**Comment by ysusuk** [comment]
benchmark shows better results as well

File: core/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala [hunk 1]
```diff
@@ -20,18 +22,24 @@ object ExpandImportSelectors extends Rewrite {
                 }
             }.flatten
             val patchStr = expandedImport.mkString("\n")
-            Patch(`import`.parent.get.tokens.head,
-                  `import`.parent.get.tokens.last,
-                  patchStr)
+
+            //TODO move up this side effect
+            `import`.parent.get.tokens.tail.foreach { tok =>
```
Comment (olafurpg) [inline core/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala]: It would be nice to add helper utilities for common operations like this. For example, `Patch.remove(Tree/Tokens)`

We can leave that for a separate PR!

**Comment by olafurpg** [comment]
Whoops, the benchmark screenshots didn't appear at first. Now I see them, sweet 😎

File: core/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala (hunk 1 above)
Comment (ysusuk) [inline core/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala]: agree!
